pkg/component/networking/vpn/shoot/shoot.go

```diff
@@ -87,8 +99,10 @@ type Values struct {
 	VPAUpdateDisabled bool
 	// ReversedVPN contains the configuration values for the ReversedVPN.
 	ReversedVPN ReversedVPNValues
-	// SeedPodNetwork is the pod CIDR of the seed.
-	SeedPodNetwork string
+	// SeedPodNetworkV4 is the v4 pod CIDR of the seed.
```

Comment:
```suggestion
	// SeedPodNetworkV4 is the IPv4 pod CIDR of the seed.
```

Reply:
Renamed back to `SeedPodNetwork`, fixed.